polkadot/node/network/availability-distribution/tests/availability-distribution-regression-bench.rs

```diff
@@ -16,7 +16,7 @@
 
 //! availability-read regression tests
 //!
-//! TODO: Explain the test case after configuration adjusted to Kusama
+//! Simulated process of availability reading on Kusama
```

Comment:
I'd go for something like `Availability read benchmark based on Kusama parameters and scale.`